### DISCUSSION TITLE
Use yaml Issue Templates

### DISCUSSION THREAD

**Discussion by capoaira** [2Abendsegler/GClh#1838] [Ideas]
Ich habe heute eine (neue?) Möglichkeit gefunden Issue Templates zu erstellen. Mit YAML ist es möglich ein Formular mit Pflichtfeldern zu erstellen:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
Als Test habe ich so etwas für das Bugreport Template bei einer anderen Repo gemacht. Wie das dann später aussieht, könnt ihr [hier](https://github.com/capoaira/GCWizard-Extension/issues/new/choose) angucken. Ich habe dabei quasi unser aktuelles Template in ein Formular umgewandelt.

Ich finde das voll geschriebene Textfeld unübersichtlich, aber ein Formular bietet mehr Übersicht und Möglichkeiten.
Was meint ihr?

**Comment by 2Abendsegler** [comment]
Ja, coole Sache. Die yml Endungen sind mir beim Sponsor Button schon mal über den Weg gelaufen, wusste aber nicht was wirklich dahinter steckt. Wenn du magst kannst du das gerne umstellen. Vielleicht zuerst mal das Template zu den Bugs und wenn das so weit in Ordnung ist, können auch die anderen umgestellt werden.

**Reply by capoaira** [reply]
Ich habe das Bug Report Template gemerged. Wir können es jetzt testen/nutzen.

**Comment by 2Abendsegler** [comment]
Ich habe das Bug Report Template bereits für #1845 genutzt. Gefällt mir ziemlich gut. 
Ich habe nur einen Punkt. Ich überlege ob wir die Info "_No response_" benötigen. Die Überschriften sind ja vorhanden und man kann für gewöhnlich sofort erkennen, dass dazu keine Info vorhanden ist. 

@capoaira, @Ruko2010, @LittleJohn-DD 
Was meint ihr?

![grafik](https://user-images.githubusercontent.com/22332216/125998768-67395dd5-365f-49a2-a7c0-704acbbf3c4a.png) &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; ![grafik](https://user-images.githubusercontent.com/22332216/125998716-dd9516e7-eb5f-4ff6-bf06-e6c2fa801b38.png)

**Reply by capoaira** [reply]
Ich denke nicht, dass wir das brauchen. Ich weiß aber auch nicht ob man das abstellen kann.

**Reply by 2Abendsegler** [reply]
Deine Bedenken scheinen angebracht. Ich konnte keine Möglichkeit finden es abzustellen.

Damit erübrigt sich auch meine Frage.  
Ich glaube ich finde es jetzt mit "_No response_" doch besser. 🤭

Aus meiner Sicht ist es also prima.

**Comment by capoaira** [comment]
Ich habe einen PR mit dem Feature Request Form erstellt. Und anstatt des "Question Reports" gibt es nun einen Link zu GitHub Discussions

**Reply by 2Abendsegler** [reply]
Nur eine Kleinigkeit: Es könnte doch auch sein, dass mein Feature Request nichts mit einem Problem zu tun hat. Wenn das Feld vorgeschrieben ist, dann weiß ich nicht so recht was ich da eintragen soll. Wieso hast du es als required Feld gemacht? 

Ansonsten finde ich beides super. Schön finde ich auch, dass man von einem angewählten Typ wieder zurückspringen kann und auch dass ein Label schon vorbelegt wird. Ja ... echt cool!

**Reply by capoaira** [reply]
Gute Frage. Das kann man aber ganz einfach ändern. Unter `validations` `required` auf `false` setzten. Soll ich das noch anpassen?

**Comment by 2Abendsegler** [comment]
Ja, wenn nichts dagegen spricht, mach das bitte noch.

**Reply by capoaira** [reply]
Von mir aus spricht nichts dagegen. => #1853

**Comment by 2Abendsegler** [comment]
Sieht alles echt super aus. Klasse gemacht!

**Comment by 2Abendsegler** [comment]
Habe Browser Auswahl um Vivaldi ergänzt.
Und habe den Text "Version" durch "GClh Version" ersetzt. Hier haben manche die Browser Version eingetragen.

@capoaira 
Bitte schau noch mal drüber.

**Reply by capoaira** [reply]
Jip, sieht gut aus. 👍🏻

**Reply by 2Abendsegler** [reply]
Was mir gerade einfällt ... Ich hatte dich irgendwann mal befördert 😄 (Config Thanks Seite) ... hatte ich ganz vergessen mal zu erwähnen. 🤭

**Reply by capoaira** [reply]
Cool, danke 😎😊
